Makefile

```diff
@@ -110,6 +110,7 @@ clean: assert-dom0 prep-salt ## Destroys all SD VMs
 	$(MAKE) destroy-all
 	sudo qubesctl --show-output --skip-dom0 --targets whonix-gw-15 state.sls sd-clean-whonix
 	sudo qubesctl --show-output state.sls sd-clean-all
+	./scripts/remove-tags
```

Comment:
do you think it would make sense to invoke this script/cleanup via `sd-clean-all.sls`? The challenge is that the path in dev environment will be different than the one in staging and production environments

Reply:
I agree that seems like the better way to do it, will poke.

Reply:
Definitely fold into `clean-salt`, as @emkll suggests. If we use a separate script, then the RPM spec will need to be updated, as well. Consider using the less elegant, but more simply `xargs`-based approach outlined above.

Reply:
The RPM spec already picks up the script by wildcard.

Reply:
What I would propose is this:
1. Move `remove-tags` from `scripts` to `dom0`
2. Add it to RPM (I think that's what you meant, Conor?)
3. Call it from `sd-clean-all.sls` as we do for `update-xfce-settings` resets.
4. Remove it from `securedrop-admin` and `Makefile`.

Does that make sense?

Reply:
@eloquence Yes, those steps make sense. Moving the invocation to the salt clean-all state is sound at this point. 

Ideally we'd call it from the scripts dir in /usr/share/, but before we can easily do that, we'll need to use the RPM in all environments, including make-dev, as described in https://github.com/freedomofpress/securedrop-workstation/issues/505#issuecomment-602814288

Reply:
Done in 49916a21b7c163904d8c770092ea483a348f62ba, not tested yet.

Reply:
Tested now, seems to work as intended (this is via `securedrop-admin`):

```
          ID: remove-rpc-policy-tags
    Function: cmd.script
        Name: salt://remove-tags
      Result: True
     Comment: Command 'salt://remove-tags' run
     Started: 17:49:47.430389
    Duration: 267.274 ms
     Changes:   
              ----------
              pid:
                  29831
              retcode:
                  0
              stderr:
              stdout:
                  Removing tag 'sd-send-app-clipboard' from VM 'work'.
                  Removing tag 'sd-receive-app-clipboard' from VM 'work'.
----------
```